client/src/components/Header/Header.scss

```diff
@@ -1,26 +1,37 @@
 @import '../../scss/partials/partials';
 
 .header {
-  padding: 0 60px;
-  margin-top: 60px;
+  padding: 20px 60px;
```

Comment:
How about a lower padding top/bottom, like `10px`? This way it matches the height from the design screenshot

Reply:
Ok, I will do that. I just felt that 15px look better :-) But I am not a designer so I will stick to the design screenshot ;-)